definitions/tools/samtools_sort.cwl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "samtools sort"
+baseCommand: ["/opt/samtools/bin/samtools", "sort"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: "mgibio/samtools-cwl:1.0.0"
+
+arguments:
+  - prefix: -o
+    valueFrom: $(runtime.outdir)/$(inputs.output_filename)
+
+inputs:
+  output_filename:
+    type: string
+    default: sorted.bam
+  nthreads:
```

Comment:
for the thread input can you make it an argument and not an input? like you did for the `star_align_fusion.cwl`. that way it can be set within the CWL and not as an input. Don't need to add a `coresMin` requirement as 1 should be okay

Reply:
does this work?